helixbass/tsc-rust
# Parse semicolon

In this PR:
- parse file consisting of a single semicolon

If you want to compare against the Typescript compiler source code, I've got `v4.5.2` checked out

To test:
If you have a file named eg `tmp.tsx` in the repo root directory whose contents are just a semicolon (no trailing newline - if your editor wants to save it with a trailing newline, you can then run `truncate -s -1 tmp.tsx` to remove the final byte from the file) and run `cargo run tmp.tsx` you should see debug-logging of a single `SourceFile` whose `statements` contains a single `EmptyStatement`
